### ISSUE TITLE
Why does the test and evaluation show different results?

### ISSUE THREAD

**Issue by jhkim0759** [Arthur151/ROMP#235]
I'm training ROMP model with your code
It saves every validation process 
but, I test the results with "python -m romp.test" code I got the different result as evaluation 
I think they are same process, do you know why ?
Or did I do something wrong?

**Comment by Arthur151** [comment]
Yes, i observe the same thing, which is quite weird. Haven't figured out the reason for now.

**Comment by jhkim0759** [comment]
Thank you. I'll tell you when I find out why.

**Comment by jhkim0759** [comment]
I found that it was happened because of 'match_preds_to_gts_for_supervision' argument.
if match_preds_to_gts_for_supervision argument is true, the result shows same as training
